examples/multi-services/utils/metrics.go

```diff
@@ -0,0 +1,62 @@
+package utils
+
+import (
+	"context"
+	"time"
+
+	otelchimetric "github.com/riandyrn/otelchi/metric"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc"
+	"go.opentelemetry.io/otel/sdk/metric"
+	"go.opentelemetry.io/otel/sdk/resource"
+	semconv "go.opentelemetry.io/otel/semconv/v1.21.0"
+)
+
+// NewMetricConfig creates metric configuration that includes:
+// - Request Duration Metrics: measures the latency of HTTP requests
+// - Request Inflight Metrics: tracks the number of concurrent requests
+// - Response Size Metrics: measures the size of HTTP responses
+func NewMetricConfig(serviceName string) (otelchimetric.BaseConfig, error) {
+	// create context
+	ctx := context.Background()
+
+	// create otlp exporter
+	exporter, err := otlpmetricgrpc.New(
+		ctx,
+		otlpmetricgrpc.WithInsecure(),
+		otlpmetricgrpc.WithEndpoint("otel-collector:4317"),
```

Comment:
By default, it will use the environment variable OTEL_EXPORTER_OTLP_METRICS_ENDPOINT, similar to how it’s handled in `utils/tracer.go`. It’s better to rely on the default endpoint instead of hardcoding it directly here.

Additionally, please add a comment indicating that we are using the endpoint from the default configuration.

Reply:
hey, i've updated it using http since the grpc didn't detected the OTEL_EXPORTER_OTLP_METRICS_ENDPOINT, and made changes using pattern in utils/tracer.go. this approach ensures we're consistently using the environment variables for configuration across both tracing and metrics.